### PR TITLE
AC-870: Use of Viewbinding instead of findViewbyId calls in Dashboard fragment

### PR DESCRIPTION
**Pull request by amtheshubham**
## Description of what I changed
I have replaced `findViewById` calls by _**ViewBinding**_ in Dashboard fragment to ensure compile time safety.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-870

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**

- [ ] No new tests required

- [x] All existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt [hunk 1]
```diff
@@ -35,13 +35,16 @@ import org.openmrs.mobile.activities.addeditpatient.AddEditPatientActivity
...
 import org.openmrs.mobile.utilities.ApplicationConstants
 import org.openmrs.mobile.utilities.ImageUtils
 import org.openmrs.mobile.utilities.ThemeUtils
 import org.openmrs.mobile.utilities.ToastUtil
 
 class DashboardFragment : ACBaseFragment<DashboardContract.Presenter>(), DashboardContract.View, View.OnClickListener {
 
+    var binding: FragmentDashboardBinding? = null
+
     private var mFindPatientButton: ImageView? = null
```
Comment (LuGO0) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt:48]: should It be private access Modifier?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt [hunk 2]
```diff
@@ -112,25 +115,27 @@ class DashboardFragment : ACBaseFragment<DashboardContract.Presenter>(), Dashboa
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
-        val root = inflater.inflate(R.layout.fragment_dashboard, container, false)
+
+        binding = FragmentDashboardBinding.inflate(inflater, container, false)
+        val root = binding!!.root
```
Comment (LuGO0) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt:120]: could you reformat the file and remove extra spaces added.

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Use view binding inside `showOverlayTutorial` function too.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt (hunk 2 above)
Comment (amtheshubham) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt:119]: Done!

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt (hunk 1 above)
Comment (amtheshubham) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.kt:47]: Done!

**Review by rishabh-997 — APPROVED** [review]

**Pull request merged**